lib/libc/minimal/include/stdlib.h

```diff
@@ -18,6 +18,12 @@ unsigned long int strtoul(const char *str, char **endptr, int base);
 long int strtol(const char *str, char **endptr, int base);
 int atoi(const char *s);
 
+void *malloc(size_t size);
+void free(void *ptr);
+void *calloc(size_t nmemb, size_t size);
+void *realloc(void *ptr, size_t size);
+void *reallocarray(void *ptr, size_t nmemb, size_t size);
+
```

Comment:
FWIW: if the arena size is zero (the default, which I like!), all the allocation calls will soft fail at runtime.  Might be more helpful (albeit less standards compliant) to #if out the declarations entirely in that case so users don't get confused about why their code is failing.